spec/meilisearch/client/indexes_spec.rb

```diff
@@ -110,42 +110,42 @@
     context 'when the uid format is invalid' do
       it 'raises an error' do
         expect do
-          client.create_index('two words')
+          client.create_index('ancient books')
         end.to raise_meilisearch_api_error_with(400, 'invalid_index_uid', 'invalid_request')
       end
     end
   end
 
   describe '#indexes' do
     it 'returns MeiliSearch::Index objects' do
-      client.create_index!('index')
+      client.create_index!('books')
 
       index = client.indexes['results'].first
 
       expect(index).to be_a(MeiliSearch::Index)
     end
 
     it 'gets a list of indexes' do
-      ['first_index', 'second_index', 'third_index'].each { |name| client.create_index!(name) }
+      ['books', 'colors', 'artists'].each { |name| client.create_index!(name) }
 
       indexes = client.indexes['results']
 
       expect(indexes).to be_a(Array)
       expect(indexes.length).to eq(3)
       uids = indexes.map(&:uid)
-      expect(uids).to contain_exactly('first_index', 'second_index', 'third_index')
+      expect(uids).to contain_exactly('books', 'colors', 'artists')
     end
 
     it 'paginates indexes list with limit and offset' do
-      ['first_index', 'second_index', 'third_index'].each { |name| client.create_index!(name) }
+      ['books', 'colors', 'artists'].each { |name| client.create_index!(name) }
 
       indexes = client.indexes(limit: 1, offset: 2)
 
       expect(indexes['results']).to be_a(Array)
       expect(indexes['total']).to eq(3)
       expect(indexes['limit']).to eq(1)
       expect(indexes['offset']).to eq(2)
-      expect(indexes['results'].map(&:uid)).to eq(['third_index'])
+      expect(indexes['results'].map(&:uid)).to eq(['colors'])
```

Comment:
This looks strange, but is correct. Apparently we sort this before returning return and the last one, in this case, is `colors`. `third_index` was also the third by coincidence 😉